### PR TITLE
Do not sort getAllPersons each time. Remove unused function.

### PR DESCRIPTION
**Pull request by UgurGumushan**

### CODE REVIEW

**Comment by mbayraktr** [comment]
it is a good point 👍 we might need find the places where the sort is needed and pass sort as true there
might be `showPersons`, `listPersons` apis.

**Comment by UgurGumushan** [comment]
> it is a good point 👍 we might need find the places where the sort is needed and pass sort as true there
> might be `showPersons`, `listPersons` apis.

Yes I have already set sorting to true in those endpoints

**Comment by mbayraktr** [comment]
> > it is a good point 👍 we might need find the places where the sort is needed and pass sort as true there
> > might be `showPersons`, `listPersons` apis.
> 
> Yes I have already set sorting to true in those endpoints

ah sorry just saw it :D

**Comment by UgurGumushan** [comment]
> > > it is a good point 👍 we might need find the places where the sort is needed and pass sort as true there
> > > might be `showPersons`, `listPersons` apis.
> > 
> > 
> > Yes I have already set sorting to true in those endpoints
> 
> ah sorry just saw it :D

I still need an approval 🙂

File: src/db.ts [hunk 1]
```diff
@@ -302,25 +302,25 @@ export const saveBooking = (accountId, booking) => {
...
+    ? persons.sort((p1, p2) => {
         if (!p1.createdAt && p2.createdAt) return 1;
         if (p1.createdAt && !p2.createdAt) return -1;
         if (!p1.createdAt && !p2.createdAt) return 0;
         return p1.createdAt > p2.createdAt ? -1 : 1;
       })
-    )
-    .then((results) => results.map((person) => augmentPerson(person)));
+    : persons;
+  persons = persons.map((person) => augmentPerson(person));
```
Comment (mbayraktr) [inline src/db.ts]: we could directly return here without new assigning

Comment (UgurGumushan) [reply src/db.ts]: Yes: https://github.com/kontist/mock-solaris/pull/112/commits/19ce037f552e64969cd24e3271c0b74e8b13a10e

**Review by mbayraktr — APPROVED** [review]

**Pull request merged**